cocos/2d/renderer/render-entity.ts

```diff
@@ -182,6 +182,7 @@ export class RenderEntity {
 
     public clearStaticRenderDrawInfos (): void {
         if (JSB) {
+            this._nativeObj.staticDrawInfoSize = 0;
```

Comment:
What problems could the unmodified version cause?

Reply:
@GengineJS It's not needed since we reset it to 0 in c++.
See
```c++
void RenderEntity::clearStaticRenderDrawInfos() {
    CC_ASSERT_EQ(_renderEntityType, RenderEntityType::STATIC);

    for (uint32_t i = 0; i < _staticDrawInfoSize; i++) {
        RenderDrawInfo& drawInfo = _staticDrawInfos[i];
        drawInfo.resetDrawInfo();
    }
    _staticDrawInfoSize = 0; // --> here
}
```